friendsController.js

```diff
@@ -0,0 +1,42 @@
+const friends = require('./friends');
+const router = require('express').Router();
+
+
+
+router.get('/',(req,res) => {
+    friends.find({},(err,results) => {
```

Comment:
Make sure we're handling errors in the case that MongoDB cannot be reached.

So something like this should suffice:

```
if (!err) {
  res.status(200).json(results)
} else {
  res.status(500).json({ errorMessage: "The friends information could not be retrieved." })
}
```